addons/message_whats_new_v2.13/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.13",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.13.0",
+    "max_client_version": "2.13.9"
+  },
+  "javascript": {
+    "enable": "setDate.js"
+  },
+  "message": {
+    "date": 1675040400,
+    "id": "message_whats_new_v2.13",
+    "title": "Welcome to Mozilla VPN 2.13",
+    "subtitle": "You’re using the latest version of Mozilla VPN. Take a look at the newest features and updates:",
+    "badge": "whats_new",
+    "blocks": [
+      {
+        "id": "c_1",
+        "type": "ulist",
+        "content": [
+          {
+            "id": "l_2",
+            "content": "When running a speed test, you’ll now be able to see upload as well as download speed."
+          },
+          {
+            "id": "l_3",
+            "content": "You can now see the IP address of the server you are connected by tapping the info button."
+          },
+          {
+            "id": "l_4",
+            "content": "On Android, you can now connect and disconnect to the VPN using the quick actions in the notification bar."
```

Comment:
Another question for content: this sounds off to me (`disconnected to the VPN`)?

Reply:
cc @chrisrooneys

Reply:
```suggestion
            "content": "On Android, you can now turn the VPN on or off using the quick actions in the notification bar."
```